src/main/java/com/github/dockerjava/api/model/Statistics.java

```diff
@@ -19,6 +21,17 @@
     @JsonProperty("read")
     private String read;
 
+    /**
+     * @since Docker Remote API 1.21
+     */
+    @CheckForNull
+    @JsonProperty("networks")
+    private Map<String, Object> networksStats;
```

Comment:
@marcuslinke just create new branch based on https://github.com/docker-java/docker-java/commit/cc5b11a7407d5ab89d5e1564b6ea86f7bad63adf (hope it has no bad changes before) then @bonigarcia will PR to this branch and you will do 2.1.3 in parallel.